buildroot-external/package/pi-bluetooth/hcidisable.service

```
@@ -0,0 +1,10 @@
+[Unit]
+Description=Remove HCI kernel driver if WiFi/Bluetooth module is not present
+ConditionPathExists=!/sys/bus/sdio/devices/mmc1:0001:1
+
+[Service]
+Type=forking
+ExecStart=/usr/sbin/rmmod hci_uart
```

`rmmod` will fail with exit code 1 and message if the module is not loaded (e.g. with `krnbt=off`), `modprobe -r` exits gracefully and also removes eventual dependencies (even though there shouldn't be any in this case). 

```suggestion
ExecStart=/usr/sbin/modprobe -r hci_uart
```